tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py{37}
+    lint
+
+[flake8]
+max-line-length= 100
+exclude= tests,venv,docs
```

Comment:
We have linted `tests/` in other repos. Why skip it this time?

Reply:
:+1: copy/pasta error